libtsuba/src/RDGTopology.cpp

```diff
@@ -146,10 +146,6 @@ RDGTopology::Map() {
   // EdgeTypeAwareTopologies have a larger adj_indices array than usual topologies
   if (topology_state_ ==
       tsuba::RDGTopology::TopologyKind::kEdgeTypeAwareTopology) {
-    KATANA_LOG_VASSERT(
-        edge_condensed_type_id_map_size_,
-        "mapping an EdgeTypeAwareTopology with a zero sized "
-        "condensed_type_id_map");
     adj_indices_size = num_nodes_ * edge_condensed_type_id_map_size_;
```

Comment:
Even if the `edge_condensed_type_id_map_size_` == 0,  can we guarantee `num_nodes` is also `0`? . With just this change, there is a chance we aren't adjusting the cursor properly. 

It should read:
```  // EdgeTypeAwareTopologies have a larger adj_indices array than usual topologies
  if (topology_state_ ==
      tsuba::RDGTopology::TopologyKind::kEdgeTypeAwareTopology && edge_condensed_type_id_map_size_ != 0 ) {
    adj_indices_size = num_nodes_ * edge_condensed_type_id_map_size_;
  }
```

Reply:
I don't think we can guarantee that num_nodes is 0. It is possible to clear edges exclusively.